### DISCUSSION TITLE
Deprecate Remix ESLint Config

### DISCUSSION THREAD

**Discussion by leonardo-ornelas** [epicweb-dev/epic-stack#606] [Ideas]
According to [this](https://github.com/remix-run/remix/issues/7056) the remix guys are no longer using  "@remix-run/eslint-config" in favour of converting to simpler plain ESLint configs.

Should Epic Stack do the same?

**Comment by kentcdodds** [comment]
Yes, I should put together an `@epic-web/eslint-config` and use that.
